designs/isolate-cluster-upgrades.md

```diff
@@ -0,0 +1,221 @@
+# Isolate workload clusters from management cluster upgrades
+
+## Introduction
+When a management cluster is upgraded, it's management components are also upgraded. This includes the EKS-A controller manager. Given this controller contains the logic to "convert" the EKS-A to the cluster-api API, any changes to this logic between one EKS-A version to the next one can trigger a cascading effect in the workload clusters after upgrading the management cluster. In some scenarios, this can cause unexpected machine rollouts.
+
+## Tenets
+1. **Simple**. Prioritize (in order): simple for the user, simple to maintain, simple to implement.
+
+## Goals
+1. As a cluster operator, I want to upgrade my management clusters without new machines being rolled out in the workload clusters.
+2. As a cluster operator, I want EKS-A to fully take care of the upgrade process, I don't want to have to manually update the workload clusters before I upgrade the management cluster.
+
+## Scope
+### In
+* Design a system that prevents, working transparently to the user, new machines from being rolled out in workload clusters when the management cluster is upgraded.
```

Comment:
Perhaps, but it's also less clear IMO. Because it's not when the cluster has been upgraded, it's **just after it has been upgraded**. Not string opinion here, open to better wording.